docs/widgetbook-cloud/quick-start.mdx

```diff
@@ -25,10 +25,11 @@ This quick start guide will help you set up and use Widgetbook Cloud efficiently
 2. Install and configure the Widgetbook CLI.
 3. Ensure that your application's resources are accessible by configuring CORS settings for your backend services.
 
```

Comment:
Step 4: Here, it feels like we're mixing Hosting and review feature. Maybe put hosting also in step 3 (once they API key is fetched and the CI/CD pipeline configured, users can upload their Widgetbook to Widgetbook Cloud) or just create another stop specifically for hosting? What do you think?

Reply:
I chnaged it slightly now, let's explorer if this is better now